data/json/mapgen/standing_stones.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "name": "GROUP_STANDING_STONES",
+    "type": "monstergroup",
+    "default": "mon_null",
+    "monsters": [
+      { "monster": "mon_darkman", "freq": 100, "cost_multiplier": 1, "conditions" : ["NIGHT"] }
```

Comment:
There appears to be an issue with the ` "conditions" : ["NIGHT"]` but there is no loading error when starting a game and the syntax is the same as the one used in monstergroups.json.

Could this be an issue with the linter?